### PR TITLE
refacto-pagination

### PR DESCRIPTION
**Pull request by PLarboulette**
Hello, 

Just a bit of refacto on the Pagination component and functions

### CODE REVIEW

File: client/src/utils/pagination.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+export const handlePageChange = ({ currentTarget: input, state }) => {
```
Comment (PLarboulette) [inline client/src/utils/pagination.js:2]: I am not sure but I think this function is never called, I don't find the way to use it

Comment (PLarboulette) [reply client/src/utils/pagination.js:2]: The goal of the function is only to update the state.pageNumber, this is is already done by the  handlePageChangeSubmission function

`this.setState({ pageNumber: pageNumber }, () => {
// Callback 
}`

**Comment by tchiotludo** [comment]
Hi @PLarboulette ! 
Thanks for this ! 
Maybe I can ask you another thing ? The pagination don't change the url and I found this annoying, Can you handle this case with this PR ? :crossed_fingers:

**Comment by PLarboulette** [comment]
Hey, 

Yes, I will look that ASAP 👍

**Pull request closed without merging**